skbio/sequence/_rna.py

```diff
@@ -135,6 +135,73 @@ def degenerate_map(cls):
     def _motifs(self):
         return _motifs
 
+    @stable(as_of="0.4.0-dev")
+    def reverse_transcribe(self):
+        """Reverse transcribe RNA into DNA.
+
+        RNA sequence is assumed to be the coding strand. Uracil (U) is
```

Comment:
This doesn't make a lot of sense, the returned `DNA` is the coding strand, not really the `RNA` which is assumed to be `mRNA` in this context.